prow/plugins/help/help.go

```diff
@@ -54,7 +60,18 @@ func goodFirstIssueMsg(helpGuidelinesURL string) string {
 	return `
 	This request has been marked as suitable for new contributors.
 
-Please ensure the request meets the requirements listed [here](` + helpGuidelinesURL + "#good-first-issue" + `).
+Please ensure that the request has the following (if relevant):
+- **No barrier to entry**
+- A **clear task**
+- A priority that **isn't too high or too low** ("Goldilocks priority")
```

Comment:
I'm concerned that this is a lot of text to read and are not really actionable and will lead to folks ignoring them. :(

How about we keep this simple and include common actionable points from the issues you've linked in https://github.com/kubernetes/community/issues/5929? Something like this (feel free to rephrase to make it sound better):

```
Please ensure that the issue body includes answers to the following questions:
- Why are we solving this issue?
- What changes need to be made to the code?
- How can the assignee reach out to you for help?
```

Reply:
Another thing to note here is that the bot is used by projects other than Kubernetes as well. We don't want to impose our help wanted/good first issue guidelines to those projects. That's one of the reasons why we have `HelpGuidelinesURL` as a config option:

https://github.com/kubernetes/test-infra/blob/1eaeec73acf524839c49550b50109d9621b9c121/prow/plugins/config.go#L98

I wonder if it might make sense to create another field in the `Help` plugin config to define the message. Something similar to how we do it for the `Welcome` plugin. If not specified, it would default to the current text.

https://github.com/kubernetes/test-infra/blob/1eaeec73acf524839c49550b50109d9621b9c121/prow/plugins/config.go#L613

Reply:
cc @mrbobbytables @alisondy

Reply:
Oh I see, this is great, thanks so much @nikhita 

I'll prune down the message text to a more actionable set of items and wait for further feedback regarding adding another field in the `Help` plugin :)